src/main/java/edu/umich/ctools/sectionsUtilityTool/FriendServlet.java

```diff
@@ -107,12 +109,15 @@ private void socialLoginRestApiCall(HttpServletRequest request,
             String keySecretStr = String.format("%s:%s", key, secret);
             M_log.info(String.format("Social login guest invitation API Request for user Email: %s Firstname: %s Lastname: %s ", nonUmichEmail, nonUmichEmailFirstName, nonUmichEmailLastName));
             String encoding = Base64.getEncoder().encodeToString((keySecretStr).getBytes());
-            CloseableHttpClient httpclient = HttpClients.createDefault();
+            CloseableHttpClient httpclient= HttpClients.custom()
+                    .setDefaultRequestConfig(RequestConfig.custom()
+                            .setCookieSpec(CookieSpecs.STANDARD).build())
+                    .build();
```

Comment:
I was seeing some warning, this code fixed it  as described [here](https://stackoverflow.com/a/40697322/724238)
```
15-Jun-2020 09:16:10.591 WARNING [https-jsse-nio-8443-exec-3] org.apache.http.client.protocol.ResponseProcessCookies.processCookies Invalid cookie header: “Set-Cookie: AWSALB=0miMU14kWDXuj2dJUD8qN1fD21B5QA2n3U4nJ3iW9j8VisAewE6qZ9G53BP3lLcGzMpoW/DMKW1AW288b04y8siHYMY5e90AlmADId40q8bxiIRMIMPPn8Z84wX4; Expires=Mon, 22 Jun 2020 13:16:10 GMT; Path=/“. Invalid ‘expires’ attribute: Mon, 22 Jun 2020 13:16:10 GMT
```